src/main/java/org/spongepowered/common/event/tracking/PhaseContext.java

```diff
@@ -295,6 +303,14 @@ public boolean isComplete() {
         return Optional.empty();
     }
 
+    public Cause toCause() {
```

Comment:
I probably missed it but I somehow need to grab whatever the current cause is at the time an event should be fired.
See here: https://github.com/SpongePowered/SpongeCommon/pull/86/files#diff-feadb8e465f1fe25208169de8ca12a02R136